### PR TITLE
mir_ci: reorganize project

### PR DESCRIPTION
**Pull request by Saviq**
Place things into a more sane structure:

```
├── mir_ci
│   ├── fixtures
│   ├── interfaces
│   ├── lib
│   ├── program
│   ├── tests
│   │   ├── clients
│   │   ├── data
│   │   └── robot
│   │       ├── libraries
│   │       ├── resources
│   │       └── templates
│   └── wayland
│       └── data
└── tools
```

Normally `tests` would live outside the Python package, but we actually want them installed.

### CODE REVIEW

**Comment by Saviq** [comment]
I'm of two minds about the `lib` folder. Both `Cgroup` and `Benchmarker` could, arguably, live under `program`…

But I'll split out `Screencopy` out from `ScreencopyTracker`, and then the tracker would have a place under `lib`…

**Review by hbatagelo — COMMENTED** [review]
I wonder if `screencopy.resource` should be renamed to something else. Maybe `keywords.resource`? It started as a resource file for the Screencopy library, but now it's also about WaylandHid.

**Comment by mattkae** [comment]
> I'm of two minds about the `lib` folder. Both `Cgroup` and `Benchmarker` could, arguably, live under `program`…
> 
> But I'll split out `Screencopy` out from `ScreencopyTracker`, and then the tracker would have a place under `lib`…

What is supposed to be the distinction between `lib` and `program` here?

**Comment by Saviq** [comment]
> I wonder if `screencopy.resource` should be renamed to something else. Maybe `keywords.resource`? It started as a resource file for the Screencopy library, but now it's also about WaylandHid.

Indeed, will come up with something.

> What is supposed to be the distinction between `lib` and `program` here?

`program` would be about running things, not necessarily in testing context; `lib` about tracking… I suppose `Cgroup` should go into `program` actually.

**Review by mattkae — APPROVED** [review]
Looks good to me :+1: I'll leave it to @hbatagelo to address his final comments

**Comment by Saviq** [comment]
> I wonder if `screencopy.resource` should be renamed to something else. Maybe `keywords.resource`? It started as a resource file for the Screencopy library, but now it's also about WaylandHid.

I'm starting to think `KVM`, as in Keyboard, Video, Mouse… but would also gladly accept something else…

**Comment by hbatagelo** [comment]
`KVM` sounds good!

File: mir-ci/mir_ci/tests/robot/resources/KVM.resource
Comment (hbatagelo) [inline mir-ci/mir_ci/tests/robot/resources/KVM.resource:1]: I think the header documentation needs to be updated as well.

Comment (Saviq) [reply mir-ci/mir_ci/tests/robot/resources/KVM.resource:1]: Done.

**Review by hbatagelo — APPROVED** [review]

**Pull request merged**